app/api/v1/endpoints/exchanges.py

```
@@ -1070,6 +1070,37 @@ async def get_kucoin_prices(currencies: List[str]) -> Dict[str, float]:
         return {}
 
 
+async def fetch_exchange_balances(exchange: str, api_key: ExchangeApiKey, db: AsyncSession) -> List[Dict[str, Any]]:
```

Ruff (0.12.2)

Redefinition of unused `fetch_exchange_balances` from line 768

(F811)